index.html

```diff
@@ -0,0 +1,186 @@
+<!DOCTYPE html>
+<html lang="fr">
+  <head>
+    <meta charset="utf-8" />
+    <title>Oh My Food</title>
+    <meta
+      name="description"
+      content="Réservez facilement dans les meilleurs restaurants Parisiens avec Ohmyfood"
+    />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.2/css/all.css"
+      integrity="sha384-vSIIfh2YWi9wW0r9iZe7RJPrKwp6bG+s9QZMoITbCckVJqGCCRhc+ccxNcdpHuYu"
+      crossorigin="anonymous"
+    />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Shrikhand&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <!-- add this font on css font-family: 'Shrikhand', cursive,'Roboto', sans-serif; -->
+    <!--HEADER-->
+    <!--HEADER-->
+    <header>
+      <h1><a href="#goToFoot" id="goToTitle">ohmyfood</a></h1>
```

Comment:
What is this href ?? id is not needed here

Reply:
c'est pour faire une ancre vers le footer et du footer vers le header